flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/nodes/exec/stream/MatchRecognizeJsonPlanTest_jsonplan/testMatch.out

```diff
@@ -131,10 +131,12 @@
         "kind" : "CALL",
         "syntax" : "BINARY",
         "internalName" : "$$1",
+        "sqlKind" : "PATTERN_CONCAT",
```

Comment:
regarding such kind of operator, do we need version them? 
on the other hand, I think it may be better to introduce a special format of InternalName, instead of a new optional field.